src/components/ItaliaTheme/Blocks/Listing/GridGalleryTemplate.jsx

```diff
@@ -77,6 +77,8 @@ const GridGalleryTemplate = ({
                         loading="lazy"
                         role="presentation"
                         aria-hidden="true"
+                        useOriginal={false}
+                        maxSize={450}
```

Comment:
io qui metterei 500, perchè il primo blocco, se lo guardo qui https://io-comune.agamar.redturtle.it/argomenti ha altezza 500px

Reply:
quello considera la larghezza
non ci sono miniature intermedie, per cui va bene uguale, ma  450 era già arrotondato in eccesso rispetto alla larghezza massima che raggiungono quelle immagini in pagina